go.mod

```diff
@@ -6,7 +6,7 @@ toolchain go1.22.0
 
 require (
 	github.com/Masterminds/sprig/v3 v3.2.3
-	github.com/conduitio/conduit-commons v0.1.2-0.20240416084352-3f77773ddcfc
+	github.com/conduitio/conduit-commons v0.2.0
 	github.com/conduitio/conduit-connector-protocol v0.5.1-0.20240408121719-ffe7a46af296
```

Comment:
Oh I realized we also made changes in the protocol, can we relase that as well please? 🙏